pkg/jobs/registry.go

```diff
@@ -1812,6 +1815,23 @@ func (r *Registry) CheckPausepoint(name string) error {
 	return nil
 }
 
+func (r *Registry) RelocateLease(
+	ctx context.Context,
+	txn isql.Txn,
+	id jobspb.JobID,
+	destID base.SQLInstanceID,
+	destSession sqlliveness.SessionID,
```

Comment:
Should this function take a sqlinstance.InstanceInfo (or some other type that bundles the instance id and session ID)?

Reply:
InstanceInfo has a lot more going on than I need/have. I don't know if we have anything that is just the pair of id and session. I'm not sure I feel like they need to be paired?